src/lib/content.js

```diff
@@ -7,6 +7,8 @@ import fastGlob from 'fast-glob'
 import normalizePath from 'normalize-path'
 import { parseGlob } from '../util/parseGlob'
 import { env } from './sharedState'
+import log from '../util/log'
+import micromatch from 'micromatch'
```

Comment:
do we use `micromatch` anywhere else or should this be `picomatch`? Basically should match what fast-glob uses I think right?

Reply:
ah we use `micromatch` elsewhere 👍